components/src/core/Hero/Hero.tsx

```diff
@@ -73,27 +74,29 @@ const Root = styled(Box, { name: 'hero', slot: 'root' })<
         height: typeof iconSize === 'number' ? Math.max(44, iconSize) : iconSize,
         width: typeof iconSize === 'number' ? Math.max(44, iconSize) : iconSize,
         backgroundColor: iconBackgroundColor,
-    },
-    [`& .${heroClasses.values}`]: {
-        display: 'flex',
-        alignItems: 'center',
-        color: theme.palette.text.primary,
-        lineHeight: 1.2,
-        maxWidth: '100%',
-        overflow: 'hidden',
-        fontSize: '1.25rem',
-    },
-    [`& .${heroClasses.label}`]: {
-        fontSize: 'inherit',
-        lineHeight: 1.2,
-        letterSpacing: 0,
-        fontWeight: 600,
-        width: '100%',
-        textAlign: 'center',
-        overflow: 'hidden',
-        textOverflow: 'ellipsis',
-        whiteSpace: 'nowrap',
-    },
+    })
+);
+
+const Values = styled(Box, { name: 'hero', slot: 'values' })<Pick<HeroProps, undefined>>(({ theme }) => ({
```

Comment:
You should be able to remove the Hero props <pick> type definition if you aren't passing in any props